roles/installer/templates/postgres.yaml.j2

```diff
@@ -37,6 +37,26 @@ spec:
       imagePullSecrets:
         - name: {{ image_pull_secret }}
 {% endif %}
+      initContainers:
+        - name: database-check
+          image: '{{ _init_container_image }}'
+          resources:
+            requests:
+              memory: "64Mi"
+              cpu: "250m"
```

Comment:
This seems to be the reason.

```
$ kubectl get pod automation-job-1-lj2kf -o json | jq '.status.conditions[0].message'
"0/1 nodes are available: 1 Insufficient cpu."
```

I'm going to play around with tweaking the requests in [`awx_cr_molecule.yml.j2`](https://github.com/ansible/awx-operator/blob/3c2405f304e3ef4a987b81e9e1c4c8cf50ac0e6f/molecule/default/templates/awx_cr_molecule.yml.j2), but they are already pretty low.